packages/m/mame/package.yml

```diff
@@ -29,19 +29,18 @@ builddeps  :
     - pkgconfig(libjpeg)
     - pkgconfig(libpulse)
     - pkgconfig(libutf8proc)
-    - pkgconfig(lua)
+    - pkgconfig(lua-5.4)
```

Comment:
If you're not building with lua you don't need the dep

Reply:
Fixed